product_docs/docs/pgd/3.7/bdr/release-notes.mdx

```diff
@@ -5,6 +5,52 @@ originalFilePath: release-notes.md
 
 ---
 
+## BDR 3.7.21 (2023 May 16)
+
+This is a maintenance release for BDR 3.7 which includes minor
+improvements as well as fixes for issues identified in previous
+versions.
+
+Check also release notes for pglogical 3.7.21 for resolved issues
+which affect BDR as well.
+
+
+### Resolved Issues
+
+- Fix join and replication when SDW and standby_slot_names are set (BDR-3176, RT89702, RT89536)
+
+- Fix upgrades for nodes with CRDTs (BDR-3302)
+
+- Fix replication for subscriber-only node (BDE-3204, RT89814)
+
+- Fix issue where a node can be inconsistent with the group after rejoining
+  If a node had been part of a sub-group, then parted and then rejoined to the
+  group, it could be inconsistent with the group. The changes from some nodes
+  of the group would be replayed from a wrong starting point, resulting in
+  potential data-loss.
+
+- Free memory consumed by dlist node holding MQConn (BDR-3409, RT91830)
```

Comment:
Can we maybe make this "Fix memory leak in consensus process", this is supposed to be consumable for customer, they have no idea what MQConn or dlist are :)

Reply:
Also this one is much more important than SDW so it should be first

Reply:
(This is also true for 4.3.1 and 5.1.0)